Warning message if 'go-md2man' is not yet installed

Taeung opened the pull request:
If 'go-md2man' is not installed,
an error can occur when running md2man-all.sh like below:

```
  $ ./man/md2man-all.sh -q
  ./man/md2man-all.sh: line 21: go-md2man: command not found
```
So fix it.

cyphar commented on man/md2man-all.sh:
```diff
@@ -9,6 +9,12 @@ cd "$(dirname "$(readlink -f "$BASH_SOURCE")")"
 }
 
+has_md2man=`which go-md2man > /dev/null 2>&1 && echo yes || echo no`
+if [ $has_md2man == "no" ]; then
```
I'd prefer something more like this:

```
if ( which go-md2man &>/dev/null ); then
  # ...
fi
```

As it's cleaner.

Taeung replied:
Hi @cyphar  😄 

Thanks for your advice, I changed it just now !

Taeung commented:
I modified this PR based on your comment ! 😄 
and tested it in my own way, I'd appreciate it, if you check this.

(In addtion, I fixed the indent size)

Taeung commented:
Hi @cyphar 
I'd appreciate it, if you recheck this PR when time allows 😄

cyphar commented on man/md2man-all.sh:
```diff
@@ -9,14 +9,19 @@ cd "$(dirname "$(readlink -f "$BASH_SOURCE")")"
-	mkdir -p "./man${num}"
-	go-md2man -in "$FILE" -out "./man${num}/${name}"
-done
+if ( which go-md2man &>/dev/null ); then
```
Sorry for not responding. Can you please simplify this change by just adding 

```
if ! ( which go-md2man &>/dev/null ); then
	echo "To install man pages, please install 'go-md2man'."
	exit 0
fi
```

And then leave the rest of the script unchanged?

Taeung replied:
OK I got it !! :smile:

Taeung commented:
I changed it based on your comment, @cyphar 😸

cyphar commented:
LGTM, thanks @Taeung!

Taeung commented:
Thanks @cyphar @TomSweeneyRedHat 😸

cyphar commented:
/ping @opencontainers/runc-maintainers

hqhq commented:
LGTM
